src/coreclr/jit/gentree.cpp

```diff
@@ -6212,6 +6230,176 @@ unsigned Compiler::gtSetEvalOrder(GenTree* tree)
 #pragma warning(pop)
 #endif
 
+//------------------------------------------------------------------------
+// gtSetEvalOrderMinOpts: A MinOpts specific version of gtSetEvalOrder. We don't
+//    need to set costs, but we're looking for opportunities to swap operands.
+//
+// Arguments:
+//    tree - The tree for which we are setting the evaluation order.
+//
+// Return Value:
+//    the Sethi 'complexity' estimate for this tree (the higher
+//    the number, the higher is the tree's resources requirement)
+//
+unsigned Compiler::gtSetEvalOrderMinOpts(GenTree* tree)
+{
+    assert(tree);
+    if (fgOrder == FGOrderLinear)
+    {
+        // We don't re-order operands in LIR anyway.
+        return 0;
```

Comment:
Does it ever get called?

Reply:
> Does it ever get called?

Not sure, but the original function `gtSetEvalOrder` contained this check